docusaurus/docs/React/guides/sdk-state-management.mdx

```diff
@@ -0,0 +1,146 @@
+---
+id: sdk-state-management
+title: SDK State Management
+---
+
+Most of the application state which is the main driver behind the UI changes of our chat SDK lives within our LLC ([`StreamChat`](https://github.com/GetStream/stream-chat-js)) - more specifically client and channel instances and is _NOT REACTIVE_ - our React SDK makes these state stores reactive by listening to the same events client does and copies it with the help of `useState` API after it has been changed.
```

Comment:
If I'm starting here, I probably don't even know what LLC means. Let's start with a less technical overview, and dive deeper later in the article?